tests/ignite/contrib/metrics/regression/test_canberra_metric.py

```diff
@@ -63,6 +64,42 @@ def test_compute():
     assert canberra.pairwise([v1, v2])[0][1] == pytest.approx(np_sum)
 
 
+def test_integration_binary_input_with_output_transform():
```

Comment:
This kind of granularity "_with_output_transform" looks strange. Why do we test only with `output_transform` and not without ?

Reply:
I always keep "with_output_transform" separated from "without_output_transform", i thought `test_compute` does testing "without_output_transform" part, i can add "without_output_transform" tests if needed..